lib/sensu-plugins-postgres/pgpass.rb

```diff
@@ -0,0 +1,17 @@
+module Pgpass
+  def pgpass
+    if File.file?(config[:pgpass])
+      pgpass = Hash[[:hostname, :port, :database, :user, :password].zip(File.read(config[:pgpass]).strip.split(':'))]
+      pgpass[:database] = nil if pgpass[:database] == '*'
```

Comment:
Makes sense to me from a quick re-read of the code, I think we should add the precedence of options to the readme so that it is clear how they can be used.